addons/digest/data/digest_data.xml

```diff
@@ -10,330 +10,252 @@
             }
             #footer {
-                background-color: #eeeeee;
-                color: #8f8f8f;
-                text-align: center;
+                font-size: 15px;
             }
-
-            @media only screen and (max-width: 650px) {
-                .d-block {
-                    display: block !important;
-                }
-                #header_background {
-                    padding-top: 0px;
-                }
-                #header {
-                    padding: 15px 20px;
-                    border: 1px solid #eeeeee;
-                }
-                .global_layout {
-                    padding: 20px;
-                }
-                .company_name {
-                    font-size: 15px;
-                }
-                .title_subtitle {
-                    margin: 5px auto;
-                }
-                #button_connect {
-                    padding: 4px 10px;
-                    font-size: 12px;
-                }
-                .date {
-                    margin-top: 10px;
-                    font-size: 10px;
-                }
-                .tip_title {
-                    font-size: 14px;
-                }
-                .tip_content {
-                    margin: 10px auto 0 auto;
-                    font-size: 12px;
-                    line-height: 20px;
-                }
-                .illustration_border {
-                    margin-top: 15px;
-                }
-                .kpi_value {
-                    font-size: 20px;
-                }
-                .kpi_margin {
-                    font-size: 10px;
-                }
-                .kpi_margin_margin {
-                    margin-bottom: 5px;
-                }
-                .preference_div {
-                    padding: 15px;
-                }
-                .preference {
-                    font-size: 12px;
-                }
-                .by_odoo {
-                    font-size: 10px;
-                }
-                .run_business {
-                    margin: 20px auto;
-                    font-size: 12px;
-                }
-                #footer {
-                    font-size: 15px;
-                }
-                #powered {
-                    margin-top: 15px;
-                }
-                .tip_twocol {
-                    overflow: auto;
-                    margin-top: 15px;
-                }
-                .tip_twocol_left {
-                    text-align: left;
-                }
-                .tip_twocol_img {
-                    width: 90%;
-                }
+            #powered {
+                margin-top: 15px;
             }
-
-            @media only screen and (min-width: 651px) {
-                #header_background {
-                    padding-top: 70px;
-                }
-                #header {
-                    padding: 20px 30px 25px 30px;
-                    border-start: 1px solid var(--color-company);
```

Comment:
As discussed on Discord, the team in charge of the bootstrap change will update the runbot shortly (if not already) for the ``border-left`` and ``border-right`` issue, so we can set them again to avoid removing it for no good reason :)

Reply:
Done